plugins/basestate/include/yarp/dev/BaseStateDriver.h

```diff
@@ -49,9 +49,10 @@ namespace yarp
 /**
  * YARP device driver to give the gazebo state information of a desired base link of the robot
  * The state information include 
- * - absolute pose of the base link (x, y, z. roll, pitch, yaw)
+ * - absolute pose of the base link (x, y, z. roll, pitch, yaw) 
  * - linear and angular velocity of the base link in the world frame
  * - linear and angular acceleration of the base link in the world frame
+ * \warning Note that the angles are expressed in radians which is contrary to the usual YARP convention of expressing angles in degrees
```

Comment:
```suggestion
 * \warning Note that the position angles, angular velocity and angular accelerations are expressed in radians-based units which is different from the usual YARP convention of expressing angles in degrees
```

Reply:
Modified in https://github.com/robotology/gazebo-yarp-plugins/commit/0c9c3cde184bcf9192e51d61ddc96da404ace358